scarb/src/bin/scarb/commands/update.rs

```diff
@@ -0,0 +1,12 @@
+use anyhow::Result;
+
+use scarb::core::Config;
+use scarb::ops;
+use scarb::ops::ResolveOpts;
+
+#[tracing::instrument(skip_all, level = "info")]
+pub fn run(config: &Config) -> Result<()> {
+    let ws = ops::read_workspace(config.manifest_path(), config)?;
+    let opts = ResolveOpts { update: true };
+    ops::resolve_workspace_with_opts(&ws, &opts).map(|_| ())
```

Comment:
nit, this looks a bit more readable to me

```suggestion
    ops::resolve_workspace_with_opts(&ws, &opts)?;
    Ok(())
```